src/fragments/lib/in-app-messaging/js/display-message.mdx

```diff
@@ -0,0 +1,33 @@
+In-app messages are displayed when an In-App Messaging or analytics event is sent and matches the criteria set forth by your active In-App Messaging campaigns.
+
+## Analytics event
+
+Now that messages have been synced to your users’ devices, Amplify In-App Messaging will allow you to start displaying them with Amplify Analytics events with no additional integration steps. Any events you record or are already recording using the `Analytics.record` API are automatically picked up and processed by In-App Messaging. If the event matches the attributes and criteria defined in an in-app message, that message will be displayed.
```

Comment:
Now that messages have been synced to your users’ devices, Amplify In-App Messaging allows you to start displaying them through Amplify Analytics events with no additional integration steps.